apps/application-system/api/src/environments/environment.interface.ts

```diff
@@ -8,6 +8,7 @@ import { SyslumennClientConfig } from '@island.is/api/domains/syslumenn'
 export interface Environment {
   production: boolean
   environment: string
+  name: string
```

Comment:
I used `name` since it exists on all the pods with a different name for each environment

![image](https://user-images.githubusercontent.com/6312838/111778820-2ff16100-88ad-11eb-9ff4-53000e0447ff.png)

![image](https://user-images.githubusercontent.com/6312838/111778768-1bad6400-88ad-11eb-82ce-68727275695f.png)

Reply:
I see, make sense 👍